src/renderer/pages/Apps.tsx

```diff
@@ -238,13 +253,29 @@ export default function Apps() {
           } else if (installedApps[app.id]) {
             if (loading[app.id]) {
               appRightJsx = <Loader />;
+              } else if (appConfig?.stableNodeVersion !== nodeConfigRelease) {
+                appRightJsx = (
+                  <>
+                    <FontAwesomeIcon
+                      onClick={() => handleAppStart(app.id)}
+                      icon={faPlay}
+                      style={{ cursor: 'pointer' }}
+                    />
+                    <FontAwesomeIcon
```

Comment:
Can we make the icon yellow? Also, it doesn't seem to work me, can you confirm on your side?

Reply:
also `appConfig?.stableNodeVersion !== nodeConfigRelease` check should be inside the JSX. So there's now else if, you simple do an else with something like this

```
<>
                    <FontAwesomeIcon
                      onClick={() => handleAppStart(app.id)}
                      icon={faPlay}
                      style={{ cursor: 'pointer' }}
                    />
                    {appConfig?.stableNodeVersion !== nodeConfigRelease && <FontAwesomeIcon
                    onMouseEnter={handleWarningIconHover}
                      onMouseLeave={handleWarningIconLeave}
                      icon={faExclamationCircle}
                      style={{ cursor: 'pointer', marginLeft: '15px' }}
                    />}
</>
```